test/core/xds/xds_cluster_resource_type_test.cc

```diff
@@ -1653,7 +1657,7 @@ TEST_F(TelemetryLabelTest, MissingMetadataField) {
   ASSERT_TRUE(decode_result.resource.ok()) << decode_result.resource.status();
   auto& resource =
       static_cast<const XdsClusterResource&>(**decode_result.resource);
-  EXPECT_EQ(resource.telemetry_labels, nullptr);
+  EXPECT_TRUE(resource.telemetry_labels.empty());
```

Comment:
Prefer to write this as `EXPECT_THAT(resource.telemetry_labels, ::testing::ElementsAre())`, since that will yield more useful output if the test fails.

Reply:
done